### PR TITLE
added

### PR DESCRIPTION
**Pull request by ZamisVlad**

### CODE REVIEW

File: main_test.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import "testing"
+
+func TestIsValidNumber(t *testing.T) {
+	tests := []struct {
+		name string
+		num  string
+		base int
+		want bool
+	}{
+		{
```
Comment (zayac) [inline main_test.go:12]: Can you please add a test with an empty string (and a valid base)?
